openmicroscopy/openmicroscopy
# Ij insight fixed develop

To test:
- Add insigh-ij plugins to plugins folder
- Add either loci_tools.jar or bioformats_package.jar to the plugins directory.
- A warning should be displayed if none present.

Check that 
- No B-F associated jar and logback under insight-ij/libs directory.

--rebased-from #2074 
--rebased-from #2060
--rebased-from #2119
